ports/stm/common-hal/audiopwmio/PWMAudioOut.c

```diff
@@ -0,0 +1,350 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 Artyom Skrobov for Adafruit Industries
```

Comment:
We use "for Adafruit Industries" to designate work that was sponsored by Adafruit, so you don't have to include it in your byline.